diff --git a/database/databaseConnection.js b/database/databaseConnection.js
index 5c360a7..c3d1ed7 100644
--- a/database/databaseConnection.js
+++ b/database/databaseConnection.js
@@ -9,4 +9,4 @@ var connect = {
 	})
 };
 
-exports.connect = connect;
+exports.connect = connect;
\ No newline at end of file
diff --git a/public/stylesheets/style.css b/public/stylesheets/style.css
index 53a0d1a..ab9d9da 100644
--- a/public/stylesheets/style.css
+++ b/public/stylesheets/style.css
@@ -1,3 +1,15 @@
+h1{
+	background-color: #eee;
+	font-style: bold;
+	height: 100px;
+	padding-top: 30px;
+}
+#pieChart {
+	display: none;
+}
+#barGraph {
+	display: none;
+}
 #content{
 	width:300px;
 	background-color: #eee;
@@ -8,12 +20,10 @@
 }
 
 .more {
-    display: none;
-    width: 600px;
-	height: 600px;
-	top: 200px;
-	left: 400px;
-	border: thick solid black;
+    height: 300px;
+	width: 560px;
+	margin: 80px;
+	float: left;
 };
 
 .abc {
@@ -31,18 +41,31 @@
 	width: 50%;
     margin: 0 auto;
 }
-
 #list {
 	width: screen.width;
 	height: 700px;
 	margin: 20px;
-	border: solid;
-	float: left;	
+	float: left;
+	font-size: 20px;	
 }
 
-#mainDiv{
-	height: 700px;
-	margin: 70px;
-	border: solid;
+.login{
+	padding: 20px;
+	margin-left: 80px;
+	background-color: #eee;
+	width: 560px;
+	height: 300px;
 	float: left;
+}
+.inputBoxName{
+	padding-left: 210px;
+	font-size: 1.5em;
+}
+.submit{
+	padding: 20px;
+	margin-left: 200px;
+}
+.inputList{
+	font-size: 1.5em;
+	margin-left: 150px;
 }
\ No newline at end of file
diff --git a/views/home.jade b/views/home.jade
index ad43f6a..a81f5b3 100644
--- a/views/home.jade
+++ b/views/home.jade
@@ -1,69 +1,55 @@
 html
-  html
-    head
-      link(rel='stylesheet', href='/stylesheets/style.css', type='text/css')
-      link(rel='stylesheet', href='/stylesheets/bootstrap.min.css')
-      script(language='javascript', type='text/javascript')
-      script.
-        function showHide(shID) {
-        if (document.getElementById(shID)) {
-        if (document.getElementById(shID+'-show').style.display != 'none')
-        document.getElementById(shID).style.display = 'block';
-        }
-        }
-
-    body
-      h1
-        center
-          u Chart Gallery
-      #list
-        #wrap
-          ul.nav.nav-pills.nav-stacked(role='tablist')
-            li
-              a#barGraph-show.showLink(onclick='showHide(\'barGraph\');')
-                | Bar Graph
-            br
-            li
-              a#pieChart-show.showLink(onclick='showHide(\'pieChart\');')
-                | Pie Chart
-              li
-      #mainDiv
-        #barGraph.more
-          form(method='post', action='/barGraph', name='barGraphInputForm')
-            table.well
-              tr
-                td Table Name
-                td
-                  input(type='text', name='tableName')
-              tr
-                td X-Axis
-                td
-                  input(type='text', name='xAxis')
-              tr
-                td Y-Axis
-                td
-                  input(type='text', name='yAxis')
-              tr
-                td
-                td
-                  input(type='submit', value='Submit')
-      #mainDiv
-        #pieChart.more
-          form(method='post', action='/pieChart', name='pieChartInputForm')
-            table.well
-              tr
-                td Table Name
-                td
-                  input(type='text', name='tableName')
-              tr
-                td X-Axis
-                td
-                  input(type='text', name='xAxis')
-              tr
-                td Y-Axis
-                td
-                  input(type='text', name='yAxis')
-              tr
-                td
-                td
-                  input(type='submit', value='Submit')
+  head
+    link(rel='stylesheet', href='/stylesheets/style.css', type='text/css')
+    link(rel='stylesheet' , href='/stylesheets/bootstrap.min.css' type='text/css')
+    script.
+      function showHide(shID) {
+      if (document.getElementById(shID)) {
+      if (document.getElementById(shID).style.display!='block') {
+      var anotherInput = ((shID=='barGraph')?'pieChart':'barGraph');
+      document.getElementById(anotherInput).style.display = 'none';
+      document.getElementById(shID).style.display = 'block';
+      }
+      }
+      }
+  body
+    h1
+      center Chart Gallery
+    #list
+      #wrap
+        ul.nav.nav-pills.nav-stacked(role='tablist')
+          li
+            a#barGraph-show.showLink(onclick='showHide(\'barGraph\');')
+              | Bar Graph
+          br
+          li
+            a#pieChart-show.showLink(onclick='showHide(\'pieChart\');')
+              | Pie Chart
+    #pieChart.more
+      .login
+        form(name='inputForm', action='input.html')
+          p.inputBoxName
+            b Pie Chart
+          .inputList
+            p
+              input(type='text', name='tableName', placeholder='table name')
+            p
+              input(type='text', name='xAxis', placeholder='description column')
+            p
+              input(type='text', name='yAxis', placeholder='comparison column')
+          p.submit
+            input(type='submit', value='Submit')
+    #barGraph.more
+      .login
+        form(name='inputForm', action='barchart.html')
+          p.inputBoxName
+            b Bar Graph
+          .inputList
+            p
+              input(type='text', name='tableName', placeholder='table name')
+            p
+              input(type='text', name='xAxis', placeholder='x axis')
+            p
+              input(type='text', name='yAxis', placeholder='y axis')
+          p.submit
+            input(type='submit', value='Submit')
